swap/src/cli/command.rs

```diff
@@ -324,6 +347,24 @@ pub enum RawCommand {
         #[structopt(flatten)]
         bitcoin: Bitcoin,
     },
+    ListSellers {
+        // TODO: sane default value
+        #[structopt(
+            long,
+            help = "The peer-id of a rendezvous node that sellers register with"
+        )]
+        rendezvous_node_peer_id: PeerId,
```

Comment:
I think we should make this optional and default to asb default. CLI users shouldn't have to deal with this IMO. Its a small change.